scripts/link-check-git-diff.sh

```diff
@@ -1,13 +1,20 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-exclude="${CHECK_LINKS_EXCLUDE_LIST:-$(dirname $0)/exclude-links.txt}"
+repo="$(dirname "$(realpath "$(dirname "$0")")")"
+pushd "$repo"
+
 differ="git diff $(git merge-base HEAD origin/master)"
-changed="$differ --name-only :^redirects-list.json"
+changed="$($differ --name-only -- '*.css' '*.js' '*.jsx' '*.md' '*.tsx' '*.ts' '*.json')"
```

Comment:
so, I thought that we want to exclude the redirects list? no?

Reply:
also, WDYT about excluding tests automatically ... again, to simplify the list ... maybe can be part of the trim ticket

Reply:
hah just realised I'd blindly included `*.json` which would indeed include `redirects-list.json`.

Reply:
> also, WDYT about excluding tests automatically ... again, to simplify the list ... maybe can be part of the trim ticket

not sure exactly what you mean (e.g. CI automatically committing to exclusions?) - still probably best to discuss in #1184

Reply:
> hah just realised I'd blindly included `*.json` which would indeed include `redirects-list.json`.

fixed now

Reply:
I mean `*.test.js` - js tests that have a lot of links that probably won't ever make it the website